R/ergm.stocapprox.R

```diff
@@ -57,6 +57,12 @@ ergm.stocapprox <- function(init, nw, model,
     stats <- model$nw.stats - NVL(model$target.stats,model$nw.stats)
     target.stats <- model$target.stats
   })
+
+  control.llik <- list(MCMLE.metric=control$MCMLE.metric, MCMLE.trustregion=control$SA.trustregion,
```

Comment:
Do we want to implement a `control.ergm.llik.<METRIC>()` family of functions so that the user could potentially specify different parameters for each metric and get argument completion etc.?

Reply:
Maybe latter. At the moment the list is small and has not changed for a long while (until `ergm.tapered`/Kpenalty. At some point it may be worth adding if/when the list grows